src/ApiHandler.ts

```diff
@@ -400,7 +400,13 @@ export default class ApiHandler {
 
 		return {
 			at,
-			staking,
+			staking: {
+				stash: ledger.stash.toString(),
+				total: ledger.total.toString(),
+				active: ledger.active.toString(),
+				unlocking: ledger.unlocking.toString(),
+				claimedRewards: ledger.claimedRewards.toString(),
```

Comment:
We actually cannot assume that this is present. In some older Kusama runtimes it was `lastReward` and in older ones it didn't exist. Also `lastReward` was an integer but `claimedRewards` is an array of integers.